README.md

```diff
@@ -1,5 +1,37 @@
 # Praat: doing phonetics by computer
 
+Welcome to Praat! Praat is a speech analysis tool used for doing phonetics by computer. Praat can analyse, synthesize, and manipulate speech, and create high-quality pictures for analysis. Praat allows you to automatically apply filters based on theories of speech production to your speech recordings, so that some of the analytical work is taken off of you. Some of Praat’s most prominent features are:
+
+Speech Analysis
+
+Praat allows you to analyze different aspects of speech including pitch, formant, intensity, and voice differences. You have access to spectrograms (a visual representation of sound changing over time), and cochleagrams (a specific type of spectrogram in which different colors are associated with different sound frequencies produced by the auditory nerve).
+
+Speech Synthesis
+
+Praat allows you to generate speech based on written input. In the Praat program, you can create a source signal from scratch, or  from an existing speech signal, and you can create a filter from scratch or extract it from an existing speech signal. You can manipulate (change, adapt) both the source and the filter before doing the actual synthesis, which combines the two. (text taken from https://www.fon.hum.uva.nl/praat/manual/Source-filter_synthesis.html)
```

Comment:
Double-space in "or  from".

Reply:
Is the attribution text needed here? Otherwise maybe it could be a link to the associated manual page, although for consistency it might be nice to have similar links in the other feature-blocks.